prov/util/src/util_av.c

```diff
@@ -502,12 +502,12 @@ static int util_verify_av_attr(struct util_domain *domain,
 	case FI_AV_TABLE:
 		if ((domain->av_type != FI_AV_UNSPEC) &&
 		    (attr->type != domain->av_type)) {
-			FI_INFO(domain->prov, FI_LOG_AV, "Invalid AV type\n");
+			FI_WARN(domain->prov, FI_LOG_AV, "Invalid AV type\n");
```

Comment:
Since you are here. I suggest to print `domain->av_type` and `attr->type`

Reply:
done